packages/cosmic-proto/package.json

```diff
@@ -136,7 +144,7 @@
   "devDependencies": {
     "@agoric/cosmos": "^0.34.1",
     "@ava/typescript": "^4.1.0",
-    "@cosmology/telescope": "^1.7.1",
+    "@cosmology/telescope": "https://gitpkg.vercel.app/agoric-labs/telescope/packages/telescope?8d2c2f6ba637a5578eead09a7368dc41c262a9d0",
```

Comment:
Confirm my understanding here, please? The reason for gitpkg rather than a link to github.com is that the package we want is not at the top level?

Reply:
Ah... I see internal discussion Aug 20 to confirm that others in the shop know about gitpkg.